test/mock-rippled.ts

```diff
@@ -1,17 +1,16 @@
-'use strict'; // eslint-disable-line
-const _ = require('lodash');
-const assert = require('assert');
-const WebSocketServer = require('ws').Server;
-const EventEmitter2 = require('eventemitter2').EventEmitter2;
-const fixtures = require('./fixtures/rippled');
-const addresses = require('./fixtures/addresses');
-const hashes = require('./fixtures/hashes');
-const transactionsResponse = require('./fixtures/rippled/account-tx');
-const accountLinesResponse = require('./fixtures/rippled/account-lines');
-const accountObjectsResponse = require('./fixtures/rippled/account-objects');
-const fullLedger = require('./fixtures/rippled/ledger-full-38129.json');
-const { getFreePort } = require('./utils/net-utils');
-const fs = require('fs');
+import _ from 'lodash';
+import fs from 'fs';
+import assert from 'assert';
+import {Server as WebSocketServer} from 'ws';
+import {EventEmitter2} from 'eventemitter2';
+import fixtures from './fixtures/rippled';
+import addresses from './fixtures/addresses.json';
+import hashes from './fixtures/hashes.json';
+import transactionsResponse from './fixtures/rippled/account-tx';
+import accountLinesResponse from './fixtures/rippled/account-lines';
+import accountObjectsResponse from './fixtures/rippled/account-objects';
+import fullLedger from './fixtures/rippled/ledger-full-38129.json';
+import { getFreePort } from './utils/net-utils';
```

Comment:
nit: sometimes you use `{ foo }` and sometimes `{foo}`. 

Could you change these to be consistent? I don't care about which, but we should be consistent. 

Please correct here and in other files.